spaceflights-pandas/hooks/post_gen_project.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from kedro.templates.project.hooks.utils import (
+
+    setup_template_add_ons,
+    sort_requirements,
+)
+
+
+def main(selected_add_ons):
+    current_dir = Path.cwd()
+    requirements_file_path = current_dir / "requirements.txt"
+    pyproject_file_path = current_dir / "pyproject.toml"
+    python_package_name = '{{ cookiecutter.python_package }}'
+    example_pipeline = "{{ cookiecutter.example_pipeline }}"
+
+    # Handle template directories and requirements according to selected add-ons
+    setup_template_add_ons(selected_add_ons, requirements_file_path, pyproject_file_path, python_package_name, example_pipeline)
+
+    # Sort requirements.txt file in alphabetical order
+    sort_requirements(requirements_file_path)
+
+
+if __name__ == "__main__":
+    # Get the selected add-ons from cookiecutter
+    selected_add_ons = "{{ cookiecutter.add_ons }}"
+
+    # Execute the script only if the add-on is selected.
+    # This ensures the script doesn't run with kedro new --starter but only with the add-ons flow option.
+    if selected_add_ons != "none":
```

Comment:
@SajidAlamQB thank you! Actually, that question still struggling me, I'm not sure that that it's a correct decision, way that you did it before with `if "Pyspark" in selected_add_ons` feels more safe for me, but it's not available in that case. I feel that we put in `cookiecutter.json` actually wrong default values:
```
    "add_ons": "none",
    "example_pipeline": "no"
```
because we will save a list of add_ons for add_ons and a Bool for example_pipeline. But I cannot understand how important is it.